From b5a9374cb5443429ca6d27a0ccba98aba6951096 Mon Sep 17 00:00:00 2001
From: Peter Tworek <tworaz666@gmail.com>
Date: Tue, 24 Mar 2015 18:01:56 +0100
Subject: [PATCH 2/8] Don't pull aura_builder dependency when aura support is
 disabled.

Simple mistake. Putting the aura_builder inside use_aura condition makes
it once again possible to build non aura version of chromium.

Signed-off-by: Peter Tworek <tworaz666@gmail.com>
---
 build/gn_migration.gypi | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/build/gn_migration.gypi b/build/gn_migration.gypi
index 53e5a02..1cb0789 100644
--- a/build/gn_migration.gypi
+++ b/build/gn_migration.gypi
@@ -515,7 +515,6 @@
       'type': 'none',
       'dependencies': [
         'All',
-        'aura_builder',
         'blink_tests',
         'chromium_builder_asan',
         'chromium_builder_chromedriver',
@@ -525,6 +524,13 @@
         'chromium_gpu_builder',
         'chromium_gpu_debug_builder',
       ],
+      'conditions': [
+        ['use_aura==1', {
+          'dependencies': [
+            'aura_builder',
+          ],
+        }],
+      ],
     },
   ]
 }
-- 
2.0.5

